lib/animina_web/potential-partner.ex

```diff
@@ -0,0 +1,20 @@
+defmodule AniminaWeb.PotentialPartner do
+  @moduledoc """
+  Functions for querying potential partners.
+  """
+
+  alias Animina.Accounts
+  alias Animina.Accounts.User
+
+  require Ash.Query
+
+  @doc """
+  Gets random users by the given limit.
+  """
+  def get_random_users(limit \\ 10) do
```

Comment:
We should be able to call this function with a `current_user` argument to make sure that this user is not part of the result.

Reply:
The function now accepts a user as an argument